src/psyclone/psyGen.py

```diff
@@ -6079,6 +6360,15 @@ def __init__(self, reference_name, parent):
         super(Reference, self).__init__(parent=parent)
         self._reference = reference_name
 
+    @property
+    def ref_name(self):
```

Comment:
How about just calling this `name` or is there some reason why that's not a good idea?

Reply:
Yep,  `name` is better.